safe/impact_functions/generic/classified_polygon_people/parameter_definitions.py

```diff
@@ -0,0 +1,57 @@
+# coding=utf-8
+"""InaSAFE Disaster risk tool by Australian Aid - Parameter definition for
+Flood Vector on Building QGIS IF
+
+Contact : ole.moller.nielsen@gmail.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+"""
+
+__author__ = 'Samweli Twesa Mwakisambwe "Samweli" <smwltwesa6@gmail.com>'
+__date__ = '8/5/15'
+
+
+from safe_extras.parameters.string_parameter import StringParameter
+
+
+def area_id():
```

Comment:
All functions in this file are not being used by other code (we move it to definitions.py and get the value from exposure's keyword). We'd better them since we won't manage unused code.